README.md

```diff
@@ -103,7 +107,7 @@ $ sudo apt update
 $ sudo apt install libpistache-dev
 ```
 
-#### Ubuntu PPA (Stable)
+### Ubuntu PPA (Stable)
 
 Currently there are no stable release of Pistache published into the [stable](https://launchpad.net/~pistache+team/+archive/ubuntu/stable) PPA. However, when that time comes, run the following to install a stable package:
```

Comment:
@dgreatwood, this isn't really an issue you created, but feel free to fix this paragraph to reflect the fact that we now have stable releases in the stable PPA.